src/Microsoft.VisualBasic/tests/VB/Microsoft.VisualBasic.VB.Tests.vbproj

```diff
@@ -1,9 +1,44 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <ProjectGuid>{da95454e-5f0e-4f8a-871a-25d1ab62867a}</ProjectGuid>
-    <Configurations>netstandard-Debug;netstandard-Release</Configurations>
-  </PropertyGroup>
-  <ItemGroup>
-    <Compile Include="Dummy.vb" />
-  </ItemGroup>
-</Project>
\ No newline at end of file
+    <PropertyGroup>
+        <ProjectGuid>{da95454e-5f0e-4f8a-871a-25d1ab62867a}</ProjectGuid>
```

Comment:
indententation is wrong here

Reply:
for .csproj files we use 2 spaces instead of 4spaces

Reply:
That means manually editing because VB always puts in 4. I don't see any VS way to set 2 for VB or C#.